config/thermofloor/heatit20a.xml

```diff
@@ -0,0 +1,225 @@
+				<Value type="bool" genre="user" instance="10" index="0" label="Switch 09" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+				<Value type="bool" genre="user" instance="11" index="0" label="Switch 10" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+	</CommandClass>
+	<CommandClass id="38" name="COMMAND_CLASS_SWITCH_MULTILEVEL" version="1">
+				<Instance index="1" />
+				<Value type="byte" genre="user" instance="1" index="0" label="Level" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="255" value="0" />
+				<Value type="button" genre="user" instance="1" index="1" label="Bright" units="" read_only="false" write_only="true" verify_changes="false" poll_intensity="0" min="0" max="0" />
+				<Value type="button" genre="user" instance="1" index="2" label="Dim" units="" read_only="false" write_only="true" verify_changes="false" poll_intensity="0" min="0" max="0" />
+				<Value type="bool" genre="system" instance="1" index="3" label="Ignore Start Level" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="True" />
+				<Value type="byte" genre="system" instance="1" index="4" label="Start Level" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="255" value="0" />
+	</CommandClass>
+	<CommandClass id="49" name="COMMAND_CLASS_SENSOR_MULTILEVEL" version="1">
+				<Instance index="1" endpoint="11" />
+				<Instance index="2" endpoint="12" />
+				<Instance index="3" endpoint="13" />
+				<Instance index="4" endpoint="14" />
+				<Value type="decimal" genre="user" instance="1" index="1" label="Temperature input 1" units="C" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="-32.25" />
+				<Value type="decimal" genre="user" instance="2" index="1" label="Temperature input 1" units="C" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="-32.25" />
+				<Value type="decimal" genre="user" instance="3" index="1" label="Temperature input 1" units="C" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="-31.92" />
+				<Value type="decimal" genre="user" instance="4" index="1" label="Temperature input 1" units="C" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="-32.25" />
+	</CommandClass>
+	<CommandClass id="90" name="COMMAND_CLASS_DEVICE_RESET_LOCALLY" version="1" request_flags="5" issecured="true" innif="true">
+				<Instance index="1" />
+	</CommandClass>
+	<CommandClass id="94" name="COMMAND_CLASS_ZWAVEPLUS_INFO" version="1" request_flags="5" innif="true">
```

Comment:
This should not be required. It should be auto discovered.

Reply:
Removed